docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -85,6 +85,56 @@ There you can find links to upgrade notes for other versions too.
             </exec>
         </target>
     ```
+- *(low priority)* use redis as cache for doctrine and framework ([#930](https://github.com/shopsys/shopsys/pull/928))
```

Comment:
maybe best solution would be to join 3 commits into one commit.

Reply:
yep